content/en/blog/_posts/2021-04-22-gateway-api/2021-04-22-gateway-api.md

```diff
@@ -0,0 +1,199 @@
+
+---
+layout: blog
+title: 'Evolving Service networking with the Gateway API'
+date: 2021-04-22
+slug: gateway-api
+---
+
+**Authors:** Mark Church (Google), Harry Bagdi (Kong), Daneyon Hanson (Red Hat), Nick Young (VMware), Manuel Zapf (Traefik)
+
+The Ingress resource is one of the many Kubernetes success stories. It created an [ecosystem of implementations](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) on which many diverse projects were able to deliver their functionality to hundreds of thousands of Kubernetes clusters in a standardized and consistent way. This standardization helped users adopt Kubernetes. However, five years after the creation of Ingress, there are signs of fragmentation into different but strikingly similar CRDs and [overloaded annotations](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/). The same portability that made Ingress pervasive has also limited what it can do.
```

Comment:
```suggestion
The Ingress resource is one of the many Kubernetes success stories. It created an [ecosystem of implementations](/docs/concepts/services-networking/ingress-controllers/) on which many diverse projects were able to deliver their functionality to hundreds of thousands of Kubernetes clusters in a standardized and consistent way. This standardization helped users adopt Kubernetes. However, five years after the creation of Ingress, there are signs of fragmentation into different but strikingly similar CRDs and [overloaded annotations](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/). The same portability that made Ingress pervasive has also limited what it can do.
```